### DISCUSSION TITLE
YDP021 userdisk分区没了

### DISCUSSION THREAD

**Discussion by BennettNotFound** [PenUniverse/PenMods#210] [你问我答]
刚刷的满分版系统2.0.7
没有14号分区
mmcblk1
mmcblk1boot0
mmcblk1boot1
mmcblk1p1
mmcblk1p10
mmcblk1p11
mmcblk1p12
mmcblk1p2
mmcblk1p3
mmcblk1p4
mmcblk1p5
mmcblk1p6
mmcblk1p7
mmcblk1p8
mmcblk1p9
mmcblk1rpmb

已用内存7.41/16GB但没有可用内存？
penmods现在也安不上了显示成功了但没有

**Comment by Redbeanw44602** [comment]
如果是8G版，那是不会有14号分区的。
1.3.0支持8g，可以先进tg群手动装。

**Reply by BennettNotFound** [reply]
但我是16g版的
也没有

**Reply by Redbeanw44602** [reply]
建议重刷系统。或者，有可能并不是16g版的
